tests/test_copy.py

```diff
@@ -155,25 +154,25 @@ def test_copy_into_location(engine_testaccount, sql_compiler):
     # NOTE Other than expect known compiled text, submit it to RegressionTests environment and expect them to fail, but
     # because of the right reasons
     try:
-        acceptable_exc_reasons = {
-            "Failure using stage area",
-            "AWS_ROLE credentials are not allowed for this account.",
-            "AWS_ROLE credentials are invalid",
-        }
-        for stmnt in (copy_stmt_1, copy_stmt_2, copy_stmt_3, copy_stmt_4):
-            with pytest.raises(Exception) as exc:
-                conn.execute(stmnt)
-            if not any(
-                map(
-                    lambda reason: reason in str(exc) or reason in str(exc.value),
-                    acceptable_exc_reasons,
-                )
-            ):
-                raise Exception(
-                    f"Not acceptable exception: {str(exc)} {str(exc.value)}"
-                )
+        with engine_testaccount.connect() as conn:
+            acceptable_exc_reasons = {
+                "Failure using stage area",
+                "AWS_ROLE credentials are not allowed for this account.",
+                "AWS_ROLE credentials are invalid",
+            }
```

Comment:
I personally like to define constants before making a connection, but this is just a nit pick and not super important